chirp/drivers/retevis_rb17p.py

```diff
@@ -382,6 +353,12 @@ def get_memory(self, number):
             mem.duplex = int(_mem.rxfreq) > int(_mem.txfreq) and "-" or "+"
             mem.offset = abs(int(_mem.rxfreq) - int(_mem.txfreq)) * 10
 
+        for char in _nam.name:
```

Comment:
Is `_nam.name` not guaranteed to be a string here?

It feels like it'd be more natural to write something like `mem.name = _nam.name.replace("\xFF", "").rstrip()`

Reply:
No, it's a memobj of bitwise chars. You can str(_mem.name) here and the right thing will happen, but it's not a string itself. This pattern is typical for other drivers, for better or worse :)

That said, this is Jim's patch from the mailing list and I'm just proxying here...

Reply:
Thanks for the explanation.

I'm probably showing my ignorance as I post comments here but I'm trying to learn the ins and outs as they show up  :)

Reply:
No worries, glad someone's paying attention :)